src/render/glad/glad.c

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// Based on the OpenGL loader generated by glad 0.1.35 on Mon Mar 21 12:42:33 2022.
+// Based on the OpenGL loader generated by glad 0.1.34 on Thu Jun 22 14:40:03 2023.
```

Comment:
Why the glad version downbump?

Reply:
I used the [glad website](https://glad.dav1d.de/) and didn't notice the downbump until I was done.